src/org/opendatakit/briefcase/util/FileSystemUtils.java

```diff
@@ -62,7 +62,7 @@
 
   static final Log log = LogFactory.getLog(FileSystemUtils.class);
 
-  private static FormCacheble formCache = new NullFormCache();
```

Comment:
I wish the formCache weren’t here in FileSystemUtils, but it doesn’t bother me terribly.

Reply:
Oh, that bothers me a lot, it's on my top 10 list of things I can't wait to put my hands on! :D